### DISCUSSION TITLE
How to reset ALL flashcards' progress at once

### DISCUSSION THREAD

**Discussion by ArcticDev78** [st3v3nmw/obsidian-spaced-repetition#721] [Q&A]
- I have around 5 (flashcard) notes, in which there are a total of 200+ Flashcards (each Flashcard in the format `Question :: Answer`).
- I want to reset the progress of EVERY flashcard at once / simultaneously. The reason why I'm asking is that it's a little inconvenient to click 'Show Answer' and click the 'Reset card's progress' button on each flashcard 200+ times...

I don't know if this is already possible somehow, but if anyone shares how to do it, I'd really appreciate it! :)
Also, in case someone has already opened a similar discussion, I apologize in advance..

**Comment by janeoa** [comment]
+1

**Comment by Kalendula16** [comment]
+1 I would also like to see this feature in the plugin itself. At the moment, in order not to reset the parameters of each note separately, I use the dnGrep program. With this program, I replace the data in all notes at the same time. Sorry for my English.
![dnGREP_oJV9Sl8eTx](https://github.com/user-attachments/assets/0340cb25-d4a1-4c27-8927-d74e012f6d93)

**Comment by KyleKlus** [comment]
There is someone working on this feature in #1319 😊
